jobs/resource_job.go

```diff
@@ -74,6 +74,17 @@ type TaskDependency struct {
 	TaskKey string `json:"task_key,omitempty"`
 }
 
+// BEGIN Jobs + Repo integration preview
+type GitSource struct {
+	GitUrl      string `json:"git_url"`
```

Comment:
Add validator for this field, like [this](https://github.com/databrickslabs/terraform-provider-databricks/blob/master/mlflow/resource_webhook.go#L99)